prog/vnet/cert_nexus.rb

```diff
@@ -127,6 +138,13 @@ def before_run
     pop "certificate revoked and destroyed"
   end
 
+  def update_stack_restart_counter
+    current_frame = strand.stack.first
+    current_frame[:restarted] += 1
```

Comment:
Since the stack is loaded from the database, it will have keys as strings, not symbols.
```suggestion
    current_frame["restarted"] += 1
```